ansible/playbooks/roles/certbot_auto/tasks/main.yml

```diff
@@ -0,0 +1,41 @@
+---
+- name: gather Cloud/OS specific variables
+  include_vars: '{{ item }}'
+  with_first_found:
+    - '{{ ansible_bios_version }}-{{ ansible_distribution }}.yml'
+    - '{{ cloud_platform }}-{{ ansible_distribution }}.yml'
+
+- name: install certbot-auto
+  become: true
+  copy:
+    src: 'certbot-auto'
+    dest: '/usr/local/bin/'
+    mode: '0755'
+    owner: '{{ USER.user }}'
+    group: '{{ USER.user }}'
+
+- name: create ssl certificate
+  shell: 'certbot-auto certonly --standalone --email dev@harmony.one -d {{ domain }} -q --agree-tos'
+
+- name: change owner of certificates
+  become: true
+  file:
+    dest: '/etc/letsencrypt/{{ item }}'
+    owner: '{{ USER.user }}'
+    group: '{{ USER.user }}'
+    recurse: yes
+  with_items:
+    - 'live'
+    - 'archive'
+
+- name: setup cron task renew ssl certificate
+  cron:
+    name: renew ssl certificate
+    job: '/usr/local/bin/certbot-auto renew --quiet'
```

Comment:
ditto on using '-- to dry-run' to first check the expected result of ssl renew, and then run the actual command without --dry-run. Also, recommended using `--debug` as well for logging purposes.

Reply:
thanks. it need to delete/insert iptables rules before/after renew certificate. let me update it.

Reply:
already fixed